docs/models.rst

```diff
@@ -76,14 +72,14 @@ function.  However the foreign key has to be separately declared with the
 
     class Person(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        name = db.Column(db.String(50))
-        addresses = db.relationship('Address', backref='person',
-                                    lazy='dynamic')
+        name = db.Column(db.String(50), nullable=False)
+        addresses = db.relationship('Address', backref='person', lazy=True)
 
     class Address(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        email = db.Column(db.String(50))
-        person_id = db.Column(db.Integer, db.ForeignKey('person.id'))
+        email = db.Column(db.String(120), nullable=False)
+        person_id = db.Column(db.Integer, db.ForeignKey('person.id'),
```

Comment:
http://docs.sqlalchemy.org/en/latest/core/metadata.html#sqlalchemy.schema.Column.params.type_ Apparently it's been in there since before 0.9, but I just discovered it recently too.